### PR TITLE
Publish freesurferator to Flywheel Exchange

### PR DESCRIPTION
**Pull request by garikoitz**

### CODE REVIEW

File: gears/flywheel/freesurferator.json [hunk 1]
```diff
@@ -0,0 +1,225 @@
+{
+  "name": "freesurferator",
+  "label": "freesurferator: create anatomical ROIs for DWI, PET or fMRI in subject space.",
+  "description": "This gear takes either an existing FS zipped run, or an anatomical NIfTI file and performs all of the FreeSurfer cortical reconstruction process. Outputs are provided in a zip file and include the entire output directory tree from Recon-All. Configuration options exist for setting the subject ID and for converting outputs to NIfTI, OBJ, and CSV. FreeSurfer is a software package for the analysis and visualization of structural and functional neuroimaging data from cross-sectional or longitudinal studies. It is developed by the Laboratory for Computational Neuroimaging at the Athinoula A. Martinos Center for Biomedical Imaging. Please see https://surfer.nmr.mgh.harvard.edu/fswiki/FreeSurferSoftwareLicense for license information. This gear includes a bunch of atlases included (cerebellum, HCP, Neuropythy... and depending on the configuration options it will create many ROIs in subject space that can later be incorporated to other neuroimaging tools (we use it extensively with DWI data, namely RTP-pipeline",
```
Comment (NPann) [inline gears/flywheel/freesurferator.json:4]: @garikoitz can you fix the parenthesis typo toward the end. 
Also rephrase "This gear includes a bunch of atlases included"

File: gears/flywheel/freesurferator.json [hunk 2]
```diff
@@ -0,0 +1,225 @@
...
+      "base": "file",
+      "optional": true,
+      "type": {
+        "enum": [
+          "nifti"
+        ]
+      }
+    },
+    "freesurfer_license_file": {
+      "description": "FreeSurfer license file, provided during registration with FreeSurfer. This file will be copied to the $FSHOME directory and used during execution of the Gear. It is optional because it will be only required if the Freesurfer option is selected for 'coreg_method', if the license file is not there, the gear will fail.",
```
Comment (NPann) [inline gears/flywheel/freesurferator.json:143]: IIRC, you were using the `install_freesurfer_license` from the gear-toolkit which actually make this one optional right. Can you consolidate the lingo with the verbiage found at https://github.com/flywheel-apps/bids-fmriprep#freesurfer_license-optional ?

**Review by NPann — APPROVED** [review]
LGTM but the typo and additional comment on the freesurfer license

File: gears/flywheel/freesurferator.json (hunk 2 above)
Comment (garikoitz) [reply gears/flywheel/freesurferator.json]: I copied the text for both the license text and files exactly as it was in that file. In the readme I will copy the main explanation

**Review by garikoitz — COMMENTED** [review]
I am done with the changes in this file, I understand that now after you approve the PR I will copy the contents of this manifest and update the repo in gitlab, and I will update the README as well with the Freesurfer license file

**Review by NPann — APPROVED** [review]

**Pull request merged**